quadrat/templates/header.html

```diff
@@ -8,20 +8,10 @@
 <!-- /wp:column -->
 
 <!-- wp:column {"verticalAlignment":"center"} -->
-<div class="wp-block-column is-vertically-aligned-center"><!-- wp:navigation {"orientation":"horizontal","itemsJustification":"right"} -->
-<!-- wp:navigation-link {"label":"Home","url":"#"} /-->
+<div class="wp-block-column is-vertically-aligned-center">
+	
+<!-- wp:navigation {"orientation":"horizontal","itemsJustification":"right","source":"primary"} -->
```

Comment:
I love this API of identifying a nav area via a block attribute.

Reply:
+10000